uiautomationutilities/perf_tool/readme.md

```diff
@@ -8,148 +8,106 @@
 
 **Pipeline information:** https://dev.azure.com/IdentityDivision/IDDP/_build?definitionId=1254
```

Comment:
Is this going to live in `IDDP` for the foreseeable future? Or will this pipeline eventually move over to `ENGINEERING`?

Reply:
Am not sure, what are the ramifications of remaining in `IDDP`

Reply:
We're migrating our pipelines there.  We should look at doing that or making sure there's a task to do so.

Reply:
Will create a task to do so